profile_library/innr/RB 247 T/model.json

```diff
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-11-28T22:41:19.017339",
+  "measure_description": "Measured with utils/measure script",
```

Comment:
```suggestion
  "measure_description": "Measured with utils/measure script",
  "device_type": "light",
```

Reply:
I have added the missing device type light. Have fixed it in measure script this morning. So next run it should hopefully be added.